fmn/consumer.py

```diff
@@ -163,7 +164,11 @@ def work(self, session, raw_msg):
                 log.info("Autocreating account for %r" % username)
                 openid = '%s.id.fedoraproject.org' % username
                 openid_url = 'https://%s.id.fedoraproject.org' % username
-                email = get_fas_email(config.app_conf, username)
+                fasjson = config.app_conf['fasjson'].get('active')
```

Comment:
I don't know if we can assume that ``fasjson`` is always there, if we can't we could do something like ``app_conf.get("fas_json", {}).get("active")``

Reply:
+1 to that